src/App.jsx

```diff
@@ -14,70 +17,61 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
-
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? (
+          <>
+            {`${selectedGood} is selected`}
             <button
-              data-cy="RemoveButton"
+              onClick={() => setSelectedGood('')}
+              data-cy="ClearButton"
               type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+              className="delete ml-3"
+            />
+          </>
+        ) : (
+          'No goods selected'
+        )}
+      </h1>
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+      <table className="table">
+        <tbody>
+          {goods.map((good) => {
+            const isSelected = selectedGood === good;
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+            return (
+              <tr
+                data-cy="Good"
+                className={classNames({
+                  'has-background-success-light': isSelected,
+                })}
+                key={good}
+              >
+                <td>
+                  <button
+                    onClick={() => setSelectedGood(isSelected ? '' : good)}
+                    data-cy={
+                      isSelected ? 'RemoveButton' : 'AddButton'
+                    }
+                    type="button"
+                    className={`button ${isSelected && 'is-info'}`}
```

Comment:
let's use classNames package for such cases to avoid these strange classes
<img width="191" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/70220667/12d13136-547a-4216-9476-ace8e5dac361">

Reply:
still not fixed
<img width="653" alt="image" src="https://github.com/mate-academy/react_goods-selector-js/assets/70220667/3c95f8b4-331a-4f24-bad3-50584c921600">